spec/features/merchants/items/create_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe "Merchant Item Create page" do
+
+  it "has a form for new item, and redirects to merchant items index with new item listed" do
+    merchant1 = create(:merchant)
+    visit "merchants/#{merchant1.id}/items/new"
+
+    within('#create_item') do
+      fill_in "Name:", with: "Paul's Item"
+      fill_in "Description", with: "An item from Paul"
+      fill_in "Unit Price:", with: 1000
+      click_button "Submit"
+    end
+
+    expect(current_path).to eq("/merchants/#{merchant1.id}/items")
+    expect(page).to have_content("Paul's Item")
```

Comment:
this test could be inside a within block that looks for the content in the correct location.